generation/consolidate_config.sh

```diff
@@ -80,7 +86,9 @@ removeElement 'organization'
 removeElement 'scm'
 removeElement 'issueManagement'
 removeElement 'licenses'
-removeElement 'profiles'
+# Do not remove profiles in java-compute because the profile is used to
+# speed up native image test, see https://github.com/googleapis/google-cloud-java/pull/10827
+removeElement 'profiles' 'java-compute'
```

Comment:
> removeElement 'profiles' 'java-compute'

This reads to remove the profiles element from (only) java-compute. I know the implementation is the opposite. Can you make the input design of "removeElement" better?

Reply:
I changed the command to
```
removeElement --element 'profiles' --excludes 'java-compute'
```